Bitmex.NET.IntegrationTests/Tests/BitmexApiServiceFundingApiTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Bitmex.NET.Models;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bitmex.NET.IntegrationTests.Tests
+{
+    [TestClass]
+    [TestCategory("REST")]
+    public class BitmexApiServiceFundingApiTests : IntegrationTestsClass<IBitmexApiService>
+    {
+
+        [TestMethod]
+        public void should_return_fundings_list()
```

Comment:
That might happen due to the settings in the account.  Can you please remove the test method and the class?

Reply:
sure